androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java

```diff
@@ -172,6 +179,30 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         new TwaSharedPreferencesManager(this)
                 .writeLastLaunchedProviderPackageName(mTwaLauncher.getProviderPackage());
+
+        mPackageManager = getPackageManager();
```

Comment:
All of the new code here should go into its own method. Maybe even a method on some other class if we want to make it more easily testable.